CHANGELOG.md

```diff
@@ -11,4 +11,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added GitHub workflows.
 
-[Unreleased]: https://github.com/giantswarm/cluster-operator/tree/legacy
+[Unreleased]: https://github.com/giantswarm/cluster-operator/tree/master
```

Comment:
@kopiczko what are your thoughts on this? The change does not look right but it was necessary to keep moving.

Reply:
That's a bug. Needs to be fixed.

Reply:
Okay I spend some more time here and I don't think fix for this is worth the effort. We should always start with changelogs in the master branch. We'll have a few outliers like this one here but I don't think it's a big issue.

Reply:
I'm doing this anyway. I'll need to release architect for my other work.